utils/switcher/run-simulation.js

```diff
@@ -5,7 +5,7 @@ import {
   openMediaSource,
   standardXHRResponse,
 } from '../../test/test-helpers';
-import {Hls} from '../../';
+import {Hls} from '../../src/videojs-http-streaming.js';
```

Comment:
I think this was previously hidden, since rollup was going into a screen, seems like we need to be more explicit about our import.